lib/Unix/PID/Tiny.pm

```diff
@@ -263,7 +263,7 @@ sub pid_file_no_unlink {
         goto EXISTS;
     }
 
-    print {$pid_fh} int( abs($newpid) );
+    syswrite( $pid_fh, int( abs($newpid) ) );
```

Comment:
nice! I almost commented on this before but shied away. follow your nerd gut I reckon ;)